docs/frogbot-config.md

```diff
@@ -1,10 +1,15 @@
-[Go back to the main documentation page](https://github.com/jfrog/frogbot)
+~~[Go back to the main documentation page](https://github.com/jfrog/frogbot)
 
 # Creating the frogbot-config.yml file
 
-## Overview
+## What is the frogbot-config.yml file?
+The [frogbot-config.yml](templates/.frogbot/frogbot-config.yml) file includes configuration related to your projects, to help Frogbot scan your Git repositories.
 
-The [frogbot-config.yml](templates/.frogbot/frogbot-config.yml) file includes the configuration required for Frogbot to scan your Git repositories.
+## Is the file mandatory?
+Not all projects require the **frogbot-config.yml** file. For projects which have a simple structure, the file isn't mandatory.
+If the following conditions apply to your project, you don't have to create the file.
+1. The project has only one descriptor file (pom.xml, package.json, go.mod, etc.) 
+2. The descriptor file is at the root directory of the project 
```

Comment:
I think that it could be helpful to describe in which situations the config file is useful as well (Scanning multi directories, scanning multi repositories)